Content.Shared/_RMC14/CCVar/RMCCVars.cs

```diff
@@ -244,13 +244,13 @@ public sealed class RMCCVars : CVars
         CVarDef.Create("rmc.power_load_multiplier", 0.01f, CVar.REPLICATED | CVar.SERVER);
 
     public static readonly CVarDef<int> RMCMarinesPerSurvivor =
-        CVarDef.Create("rmc.marines_per_survivor", 20, CVar.REPLICATED | CVar.SERVER);
+        CVarDef.Create("rmc.marines_per_survivor", 10, CVar.REPLICATED | CVar.SERVER);
```

Comment:
We changed this in the live to 12 and 7 which works fine without wounds and synthvivor
```suggestion
        CVarDef.Create("rmc.marines_per_survivor", 12, CVar.REPLICATED | CVar.SERVER);
```

Reply:
Alright, then I'll change it to 15 and 7 too here